File: apzkr-pzpi-21-6-zamkovyi-anatolii/Task3-WebClient/WebApplication3/WebApplication3/Models/Athlete.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace WebApplication3.Models
{
    public class Athlete
    {
        public int AthleteId { get; set; }
        public string AthleteName { get; set; }
        public string Password { get; set; }
        public DateTime BirthDate { get; set; }
        public int PhoneNumber { get; set; }
        [ForeignKey("Coach")]
        public int CoachId { get; set; } // Зберігати ID тренера

    }
}

File: apzkr-pzpi-21-6-zamkovyi-anatolii/Task4-MobileClient/SportApp/app/src/main/java/com/example/sportapp/Model/Athlete.kt
package com.example.sportapp.Model

import com.google.gson.annotations.SerializedName

data class Athlete(
    @SerializedName("athleteId") val athleteId: Int?,
    @SerializedName("athleteName") val athleteName: String,
    @SerializedName("password") val password: String,
    @SerializedName("birthDate") val birthDate: String,
    @SerializedName("phoneNumber") val phoneNumber: String,
    @SerializedName("coachId") val coachId: Int? // CoachId є необов'язковим полем
)
data class AthleteRegistrationRequest(
    @SerializedName("athleteName") val athleteName: String,
    @SerializedName("password") val password: String,
    @SerializedName("birthDate") val birthDate: String,
    @SerializedName("phoneNumber") val phoneNumber: Int,
    @SerializedName("coachId") val coachId: Int? // CoachId є необов'язковим полем
)

data class AthleteRegistrationResponse(
    @SerializedName("message") val message: String
)
